src/components/react-json-schemaForm/GeometryField/PointField/PointField.js

```diff
@@ -10,46 +10,62 @@ const Label = ({ name, required }) => (
 );
 
 const PointField = ({
-  idSchema: { coordinates: { $ids } },
+  idSchema: { coordinates: { $id } },
   formData,
+  formData: { coordinates },
   onChange,
   t,
   schema: {
     required = [],
   },
   disabled,
 }) => {
-  const [coords, setCoords] = useState(formData.coordinates);
   const isRequired = required.includes('coordinates');
 
+  const refLatitude = useRef(null);
+  const refLongitude = useRef(null);
```

Comment:
Why using useRefs for numbers/strings?
Objects & Arrays may need refs because shallow comparision (`===`) is always false. But numbers and strings do not have this issue.